aws-integration/pipelines/logpipeline.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: telemetry.kyma-project.io/v1alpha1
+kind: LogPipeline
+metadata:
+  name: aws-cloudwatch
+spec:
+  input:
+    application:
+      namespaces:
+        system: true
```

Comment:
We should not ship system logs. I added that rather to receive some test data. Remove the entire `input` section.